File: clients/javascript/ai.js
const Vector = require('victor');

module.exports = class AI {
  constructor() {}

  step(game) {
    for(const cell of game.me.cells) {
      const distance = cell.position.distance(cell.target);
      // Is close to its end
      if (distance <= 10) {
        const target = (new Vector(0, 0))
          .randomize(new Vector(0, 0), new Vector(game.map.width, game.map.height));
        cell.move(target);
      }
    }

    return game;
  }
};

File: clients/python/ai.py
"""
This is the file that should be used to code your AI.
"""
import random
from planar import Vec2


class AI:
    def __init__(self):
        pass

    def step(self, game):
        """
        Given the state of the 'game', decide what your cells ('game.me.cells')
        should do.

        :param game: Game object
        """

        for cell in game.me.cells:
            distance = cell.position.distance_to(cell.target)

            if distance < 10:
                target = Vec2(random.randint(0, game.map.width), random.randint(0, game.map.height))
                cell.move(target)
